hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleDetails.java

```diff
@@ -214,4 +141,5 @@ protected String getTabSheetId() {
     protected String getDetailsHeaderCaptionId() {
         return SPUIComponetIdProvider.TARGET_DETAILS_HEADER_LABEL_ID;
     }
+
```

Comment:
Is this a rule, that our claseses should not have a empty row before the last '}'?
Currently I can't find a clear trend in the other classes.